Topics/Development_Process/Serverless_AWS_Lambda.md

```diff
@@ -0,0 +1,69 @@
+# Serverless Computing with AWS Lambda
+
+## Table of Contents
+
+### [Introduction](#introduction-1)
+
+### [Benefits of Serverless Computing](#benefits-of-serverless-computing-1)
+
+### [Disadvantages of Serverless Computing](#disadvantages-of-serverless-computing-1)
+
+### [AWS Lambda](#aws-lambda-1)
+
+#### [Creating Lambda Functions in the AWS Console](#creating-lambda-functions-in-the-aws-console-1)
+
+#### [Invoking Lambda Functions from the AWS Console](#invoking-lambda-functions-from-the-aws-console-1)
+
+### [Additional Resources](#additional-resources-1)
+
+## Introduction
+
+It can be hard to set up and maintain your backend services. There are a ton of hosting options to choose from, and they can get pretty pricey.
+
+Serverless computing allows you to to build and run applications without needing to manage servers. Cloud providers such as AWS, Azure, Google Cloud automatically provision servers based on demand.
+
+## Benefits of Serverless Computing
+
+As a developer, using serverless computing for your backend services has many benefits.
+
+1. **Lower cost**: With serverless computing, you only pay for what you use. Traditional cloud computing typically requires you to pay for a fixed server, which often results in you having to pay for idle time and unused space.
+2. **Scalability**: Because it is not up to the developer to allocate and maintain servers, developers do not have to worry about scaling servers up or down, as it is up to the cloud provider to do so.
+3. **Easier maintenance**: The developer/user does not need to manage their backend services, as it is up to the cloud provider to maintain the servers. With containers or other deployment options, developers are responsible for updating and maintaining them.
+4. **Deployment time**: Serverless functions finish deploying as soon as their code is updated, making them generally faster than containers, or virtual machines.
```

Comment:
It could be worth giving an example for this. Or elaborating more. Is it because of CI/CD pipelines being automatically set up by the cloud host? Or could it be from something else?

Reply:
That's a good idea, I didn't think of that! I will add a sentence explaining why they deploy so fast.